assets/src/application/ui/layout/espressoTable/TableBody.tsx

```diff
@@ -91,16 +72,18 @@ const TableBody = ({
 						cellData.hasOwnProperty('value'),
 						`Missing "value" property for table cell at ` + `row ${rowNumber} column ${colNumber}.`
 					);
-					if (isFunction(cellData.render)) {
-						return cellData.render(rowNumber, colNumber, column, cellData);
+
+					if (isFunc(cellData.render)) {
```

Comment:
Not sure why is the need to for `isFunc` here if we have it locked down with TS. After TS lock down, we can simply check for `if (cellData.render) {}` which will fail in case or `null` or `undefined` and rest of the cases should be eliminated by TS.

Reply:
updated: https://github.com/eventespresso/event-espresso-core/pull/2475/commits/c86f0458d727dbb7ad2b1267ee1032ac07ca7b9d